New package: VpinEstimation v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: VpinEstimation
- Repository: https://github.com/jinyunfan/VpinEstimation.jl
- Created by: @jinyunfan
- Version: v0.1.0
- Commit: 95ea205b4e4aee760ee35d4181f6f7e9f94a8f9b
- Git reference: main
- Description: With reference to Easley et al(2014), the volume synchronized probability of informed trading is estimated under high-frequency environment, and the index VPIN(volume synchronized probability of informed trading) is calculated.

goerz commented:
Thank you for resubmitting. This still needs a little bit more documentation in order to be registered. Packages in General are intended to be useful to a wider audience, which requires enough documentation to be able to figure out how to use the package. As a minimum, an initial registration might provide a usage example (with some explanations) in the README. Of course, a more complete [Documenter](https://documenter.juliadocs.org/stable/)-based documentation with tutorials etc. is always a good idea, and usually a requirement in the long term (for a v1.0 release) at least for more complex packages.

JuliaHubRegistrator committed:
New package: VpinEstimation v0.1.0

UUID: 98b75bef-cec7-42fd-9fd8-1608d125fef6
Repo: https://github.com/jinyunfan/VpinEstimation.jl.git
Tree: 48779900ce7da6676ee03cd1ba6716b2018d3189

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

jinyunfan commented:
Ok, I changed the version number to v0.1.0. Is it OK now? @goerz

goerz commented:
No… Didn’t you see [my comment](https://github.com/JuliaRegistries/General/pull/117292#issuecomment-2413552539)?

jinyunfan commented:
Sorry,I misunderstood. So I will add a usage example and function descriptions before submitting.
